client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -190,15 +192,25 @@ export const ApplicationDetailDrawer: React.FC<
                     {t("terms.associatedArchetypes")}
                   </DescriptionListTerm>
                   <DescriptionListDescription>
-                    {application?.archetypes?.length ?? 0 > 0 ? (
-                      <ArchetypeLabels
-                        archetypeRefs={application?.archetypes}
-                      />
+                    {application?.archetypes?.length ? (
+                      <>
+                        <Link to={getArchetypesUrl(application?.name)}>
+                          {application.archetypes.length}{" "}
+                          {t("terms.archetypes", {
+                            count: application.archetypes.length,
+                            context:
+                              application.archetypes.length > 1
+                                ? "plural"
+                                : "singular",
+                          }).toLocaleLowerCase()}{" "}
+                        </Link>
+                      </>
```

Comment:
With this change, we require a user to click a link to even see what archetypes the application is linked to.  I like the click over to the archetype table with filters applied for the application, but I don't like that you can't just look at it in the drawer.